src/OSWindow-SDL2/SDL_TextEditingEvent.class.st

```diff
@@ -138,24 +138,22 @@ SDL_TextEditingEvent >> start: anObject [
 
 { #category : #accessing }
 SDL_TextEditingEvent >> strlen [
+
 	| len |
 	len := 0.
-	12 to: 12 + 32 - 1 do:
-		[ :i |
-		(self getHandle nbUInt8AtOffset: i) = 0
-			ifTrue: [ ^ len ].
+	12 to: 12 + 32 - 1 do: [ :i |
```

Comment:
Argh. All these magic numbers are really bad!
(I know it is not related to this PR)

Reply:
I avoided to commit them multiple time but I forgot once :( should be pretty safe tho